package.json

```diff
@@ -74,6 +74,7 @@
     "react-papaparse": "^4.0.2",
     "react-tooltip": "^4.2.21",
     "sanitize.css": "^13.0.0",
+    "sharp": "^0.31.3",
```

Comment:
Looks like this library isn't used

Reply:
Thanks. I thought I was going to need next13 but it's not. I'll remove all changes

Reply:
I'm not 100% yet but I might all these changes to create an Docker image that will be later used to deploy to vercel (using vercel CLI) i need all this. TBD